change/@microsoft-teams-js-7130594c-7f2f-47d9-80ff-229561e1845c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added API version tags to `barcode`, `calendar`, `call`, `clipboard`, `mail`, `marketplace` and `media` capabilities",
```

Comment:
```suggestion
  "comment": "Added telemetry to `barcode`, `calendar`, `call`, `clipboard`, `mail`, `marketplace` and `media` capabilities",
```

Reply:
(These are customer facing notes, so "telemetry" is more meaningful to people than "API version tags"